README.md

```diff
@@ -1,5 +1,3 @@
-[![Stable Build Status](http://img.shields.io/travis/MetaModels/attribute_combinedvalues/master.svg?label=stable)](https://travis-ci.org/MetaModels/attribute_combinedvalues/branches)
```

Comment:
you can add

`[![Build Status](https://github.com/MetaModels/attribute_combinedvalues/actions/workflows/diagnostics.yml/badge.svg)](https://github.com/MetaModels/attribute_combinedvalues/actions)`

Reply:
Fixed in 45b95e0